integration-cli/docker_cli_port_test.go

```diff
@@ -148,9 +149,7 @@ func (s *DockerSuite) TestPortList(c *check.C) {
 
 	out, _ = dockerCmd(c, "port", ID)
 
-	err = assertPortList(c, out, []string{
-		"80/tcp -> 0.0.0.0:8000",
-		"80/udp -> 0.0.0.0:8000"})
+	err = assertPortRange(c, out, []int{8000, 8080}, []int{8000, 8080})
```

Comment:
Why is this change necessary, but the other `assertPortList` calls work fine?

Reply:
When you run this test multiple times, the TCP port used by the container increments (i.e.: first run its 8000, second run its 8001, etc.). I guess that this is because we pass a range (8000-8080) and the engine tries not to reuse recently used ports.